lua/glow.lua

```diff
@@ -79,22 +86,13 @@ local function open_window(cmd, tmp)
   vim.keymap.set("n", "q", close_window, keymaps_opts)
   vim.keymap.set("n", "<Esc>", close_window, keymaps_opts)
 
-  local cbs = {
-    on_input = function()
-      if tmp ~= nil then
-        vim.fn.delete(tmp)
-      end
-    end,
+  local term_opts = {
+    on_exit = cleanup,
+    width = win_width,
+    height = win_height,
   }
 
-  local chan = vim.api.nvim_open_term(buf, cbs)
-  job_id = vim.fn.jobstart(cmd, {
-    on_stdout = function(_, data, _)
-      for _, d in ipairs(data) do
-        vim.api.nvim_chan_send(chan, d .. "\r\n")
-      end
-    end,
-  })
+  job_id = vim.fn.termopen(cmd, term_opts)
```

Comment:
was hoping to not use this function anymore since it brings a really bad `Process Exited [0]` msg at the end of the preview. The `nvim_open_term` function was fixing that, but with the new glow version, that stopped working. Can we stick with the old behavior somehow?

Reply:
You're right, `Process Exited [0]` is less than ideal.  I'll see if there is a way to avoid that message at the end of the preview.  In the meantime, I'll convert this to a draft.

Reply:
@ellisonleao I believe I have an updated solution that successfully leverages `nvim_open_term` so that we can avoid `Process Exited [0]`.

@pynappo I'm also curious if this helps resolve the behavior you were seeing on Windows (or if it introduces new/different issues).  This works for me on Linux.